### PR TITLE
VACMS-15506 aging content fwb mvp with expirable content module

### PR DESCRIPTION
**Pull request by dsasser**
## Description
This PR:

- Adds the 'expirable_content' module from contrib which allows any content entity type to have a calculated expiration date.
- Adds a new message template for Full Width Banner warning notifications.
- Adds new ECA model to trigger the FWB warning notifications
- ECA model changes to correct processing logic flaw
- Adds relevant changes for theming the new template (theme hook, mime configuration, etc)
- Adds a patch to the token module to support computed fields
- Changes the Aging Content View to use the new fields and filters provided by the Expirable Content module.
- Adds a deploy hook to seed the initial expiration dates for FWB nodes.
- A ddev tweak to allow debugging from the command line in PHPStorm

Relates to #15506 

## Testing done
Tested locally using the provided script to generate FWB nodes, and send the notification emails by running cron both in the UI and from Drush.

## Screenshots

Screenshot 2024-04-25 at 12 45 10 PM
Screenshot 2024-04-25 at 12 41 58 PM
Screenshot 2024-04-25 at 12 40 40 PM
Screenshot 2024-04-25 at 12 39 37 PM

## QA steps

As a Tugboat user and Drupal admin:
1. Visit the Tugboat dashboard
2. Click the 'Terminal' link for the PHP service
3. In the window that opens, run the following: `drush scr scripts/content/VACMS-15506-generate-test-nodes.php`.
4. Visit the [Aging Content](https://pr17732-ubppbsl9qxkfbbjvvag7dlejqvwlbpts.ci.cms.va.gov/admin/structure/views/view/aging_content) View
   - [x] In the preview section, validate you see the node with title 'Expirable Content Test Node:expired'. **note you may see other nodes there and that is ok**
5. Visit the [Warning](https://pr17732-ubppbsl9qxkfbbjvvag7dlejqvwlbpts.ci.cms.va.gov/admin/structure/views/view/aging_content/edit/fwb_warn) display
   - [x] In the preview section, validate that you see a node with title 'Expirable Content Test Node:warn'. **note you may see other nodes there and that is ok**
6. Visit the [ECA model](https://pr17732-ubppbsl9qxkfbbjvvag7dlejqvwlbpts.ci.cms.va.gov/admin/config/workflow/eca) landing page.
7. Enable the first model 'Aging Content: Warn FWB' by clicking the Operations dropdown and selecting 'enable'. You should notice the page reloads automatically as the action is done via ajax.
8. Run CRON by hovering over the VA icon in the toolbar menu and selecting 'cron'. Wait for the page to reload.
9. Run CRON a second time. (the first time queued the message, the second time sends it)
10. Visit the Tugboat [dashboard](https://tugboat.vfs.va.gov/660ebb84b4e1649ad17fe163)
   - [x] Validate that under the 'Captured Mail' there exists an email with the title 'Please update or archive your Full-width banner by 2024-04-24'
11. Visit the [ECA model](https://pr17732-ubppbsl9qxkfbbjvvag7dlejqvwlbpts.ci.cms.va.gov/admin/config/workflow/eca) landing page.
12. Disable the 'Aging Content: Warn FWB' model and enable the 'Aging Content: Expire FWB' model.
13. Run CRON
14. Run CRON again
15. Visit the Tugboat [dashboard](https://tugboat.vfs.va.gov/660ebb84b4e1649ad17fe163)
   - [x]  Validate that under 'Captured Mail' there exist an email with the title 'Your Full-width banner has displayed for seven (7) days'

### Definition of Done

- [ ] Documentation has been updated, if applicable.
- [ ] Tests have been added if necessary.
- [ ] Automated tests have passed.
- [ ] Code Quality Tests have passed.
- [ ] Acceptance Criteria in related issue are met.
- [ ] Manual Code Review Approved.
- [ ] If there are field changes, front end output has been thoroughly checked.

### Select Team for PR review

- [ ] `CMS Team`
- [ ] `Public websites`
- [ ] `Facilities`
- [ ] `User support`
- [ ] `Accelerated Publishing`

### CODE REVIEW

File: .ddev/config.yaml [hunk 1]
```diff
@@ -27,7 +27,7 @@ use_dns_when_possible: true
 composer_version: "2"
 disable_settings_management: true
 web_environment:
-    - PHP_IDE_CONFIG='serverName=appserver'
+    - PHP_IDE_CONFIG=serverName=appserver
```
Comment (dsasser) [inline .ddev/config.yaml:30]: This change is needed to allow debugging from the command line via PHPStorm. PHPstorm, for whatever reason, cannot parse the single quotes, and throws an error when it receives an xdebug call from cli.

Comment (edmund-dunn) [reply .ddev/config.yaml:30]: Nice find!!

**Review by edmund-dunn — APPROVED** [review]
Everything worked per QA steps. Nice work! 
Screenshot 2024-04-30 at 11 36 16
Screenshot 2024-04-30 at 11 36 32
Screenshot 2024-04-30 at 11 53 42

**Review by omahane — APPROVED** [review]
This is great work, Daniel, and tests out perfectly.

**Comment by dsasser** [comment]
Noting that Edmund previously approved, and only the [composer.lock hash has changed](https://github.com/department-of-veterans-affairs/va.gov-cms/pull/17732/commits/cad6b64bfd021c8488413a67c27454bdff97b30f) (as part of merging in the latest changes from main and resolving composer.lock merge conflicts) since then. Merging based on this and new approval from Christian.
